MRN-Code/coinstac
# Manual File Groups w/in Collection

* **Problem:**
   1. Most file collections will consist of files chosen by the user individually or en masse. No method currently exists to allow users to upload files in this manner.
* **Solution:**
   1. Add a second option for manually selecting files for collection groups.
* **Tests:**
   1. View _Collections_ navigation item and view existing test collection.
   1. Click on _Files_ tab to add a new group to the existing collection.
   1. Click the _Add Group_ button and the _Manually Select Files_ radio button before hitting the _Add Files Group_ button to select your files.
  1. Add several files of the same type, an entire directory or a group of directories with subdirectories. Once selected, the files discovered should display in a panel.
  1. Add a group containing files with different extensions. An error message should display.
   1. Add files to an existing group by expanding the _Files_ accordion and hitting the _Add More Files to Group_ button. Duplicates should be removed and all files should get organized alphabetically.
